### PR TITLE
Add cron task to update TranslationFiles using TranslationTool

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add cron task to update TranslationFiles using TranslationTool, this task will be run automatically when modifications are pushed on `integration` and `preproduction` branches The production branch will be added once the workflow is validated And the environment for preproduction is not setup yet but you can see that it works fine for integration here https://github.com/jolelievre/TranslationFiles/actions/runs/3114348238/jobs/5050080933<br>A manual trigger will also be available and let you choose the prestashop version and environment
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| How to test?      | ~
| Possible impacts? | ~

### CODE REVIEW

File: .github/workflows/automatic_push_archives_to_bucket.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Update TranslationFiles packages Cron after branch push
```
Comment (eternoendless) [inline .github/workflows/automatic_push_archives_to_bucket.yml:1]: It's not a cron if it's triggered by an event right?
```suggestion
name: Update TranslationFiles packages after branch push
```

File: .github/workflows/manual_push_archives_to_bucket.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: Update TranslationFiles packages Cron on dispatch
```
Comment (eternoendless) [inline .github/workflows/manual_push_archives_to_bucket.yml:1]: Is this it?
```suggestion
name: Update TranslationFiles packages on manual trigger
```

File: .github/workflows/push_archives_to_bucket.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
+name: Update TranslationFiles packages Cron
```
Comment (eternoendless) [inline .github/workflows/push_archives_to_bucket.yml:1]: It's not a cron if it's not scheduled
```suggestion
name: Update TranslationFiles packages
```

**Review by eternoendless — COMMENTED** [review]
Looks good to me, only the script titles seem off.

**Comment by jolelievre** [comment]
Thanks @eternoendless indeed I renamed the scripts, it should be good now

**Review by eternoendless — APPROVED** [review]
LGTM

**Comment by eternoendless** [comment]
Thanks @jolelievre

**Pull request merged**